.github/workflows/trigger_prerelease.yml

```diff
@@ -0,0 +1,20 @@
+name: Trigger prerelease creation
+
+# This workflow triggers a prerelease creation with changelog and the release notes created by the release toolkit.
+# This workflow should be triggered merely from the default branch.
+# For more details about how to release follow https://github.com/newrelic/coreint-automation/blob/main/docs/release_runbook.md
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 12 * * 1" # Monday at 12pm UTC or 5am PT
+
+jobs:
+  prerelease:
+    uses: newrelic/coreint-automation/.github/workflows/trigger_prerelease.yaml@v1
+    with:
+      rt-included-files: go.mod,go.sum,build/Dockerfile
+    secrets:
+      bot_token: ${{ secrets.K8S_BOT_TOKEN }}
+      slack_channel:  ${{ secrets.K8S_SLACK_CHANNEL }}
+      slack_token: ${{ secrets.K8S_SLACK_TOKEN }}
```

Comment:
`K8S_AGENTS_BOT_TOKEN`, `K8S_AGENTS_SLACK_CHANNEL`, `K8S_AGENTS_SLACK_TOKEN`

Reply:
I've updated this. Sorry about that.